.github/workflows/docker-build.yaml

```diff
@@ -61,6 +61,8 @@ jobs:
               docker/Dockerfile.base
             sed -i 's/balenalib\/raspberrypi3/balenalib\/raspberry-pi/' \
               docker/Dockerfile.viewer
+            sed -i 's/\barmv7hf\b/rpi/' \
```

Comment:
This shouldn't be necessary since you're using the base image (which we patch above).

Reply:
Haven't tried the changes yet on Raspberry Pi 1 and 2 devices, but this line might be necessary.

Inside `Dockerfile.wifi-connect`, we can see that the latest `wifi-connect` archive will be downloaded.

https://github.com/nicomiguelino/Anthias/blob/bd4499e8c6f33aba0572055f2f406b7ac7a5c2a0/docker/Dockerfile.wifi-connect#L5-L8

```bash
RUN curl https://api.github.com/repos/balena-io/wifi-connect/releases/latest -sL \
    | grep -hoP 'browser_download_url": "\K.*armv7hf\.tar\.gz' \
    | xargs -n1 curl -Ls \
    | tar -xvz -C /usr/src/app/
```

Archive files that have `armv7hf` in their file name are for Raspberry Pi 3 and 4 devices, while those that have `rpi` are for Raspberry Pi 1 and 2 devices.

Reply:
Yeah, it's hardcoded ATM...